php-frontend/src/main/java/org/sonar/php/utils/collections/ListUtils.java

```diff
@@ -24,12 +24,31 @@
 import java.util.Collections;
 import java.util.List;
 import java.util.stream.Collectors;
+import javax.annotation.CheckForNull;
+import javax.annotation.Nullable;
 
 public class ListUtils {
 
   private ListUtils() {
   }
 
+  public static <T> T getLast(List<T> list) {
+    return list.get(list.size() - 1);
```

Comment:
Nitpick: This method throws an `IndexOutOfBoundsException` when the list is empty, guava would throw a `NoSuchElementException`. I don't see a big problem with it, as both are extending from `RunTimeException`